### PR TITLE
feat(components/molecule/avatar): make optional hex background of fallback name

### PR DESCRIPTION
**Pull request by quinwacca**
### Types of changes
- [x] ✨ New feature (non-breaking change which adds functionality)

### Description, Motivation and Context
Until now, name fallback has a randomly generated background color from the name.
At motor, we need to use a defined background color for every name fallback avatar.

### CODE REVIEW

**Review by jordevo — APPROVED** [review]

**Comment by andresin87** [comment]
why hex? why boolean?

File: components/molecule/avatar/src/AvatarFallbackName/index.js [hunk 1]
```diff
@@ -21,7 +22,8 @@ const MoleculeAvatarFallbackName = ({
       ? `${firstName.charAt(0)}${lastName.charAt(0)}`
       : firstName.charAt(0)
 
-  const backgroundColor = useConvertStringToHex(nameProp)
+  const stringToHexBackgroundColor = useConvertStringToHex(nameProp)
```
Comment (andresin87) [inline components/molecule/avatar/src/AvatarFallbackName/index.js]: backgroundColor = stringToHexBackgroundColor || useConvertStringToHex(nameProp)

useConvertStringToHex is a complex fn. It only needs its execution if the color is not proper declared

**Review by andresin87 — CHANGES_REQUESTED** [review]
Also declare the prop `fallbackColor` in the MoleculeAvatar for get it documented on the doc tab and remove the rest prop-spreading from the renderContent observables and declare it imperative using `fallbackColor={fallbackColor}`

File: components/molecule/avatar/src/AvatarFallbackName/index.js (hunk 1 above)
Comment (quinwacca) [reply components/molecule/avatar/src/AvatarFallbackName/index.js]: but hooks cannot be called conditionally, I'll change the hook then

**Review by andresin87 — APPROVED** [review]

**Pull request merged**